example/core_tutorial/05_slice/slice_example.cpp

```diff
@@ -78,6 +77,9 @@ void sliceExample()
     Cabana::AoSoA<DataTypes, DeviceType, VectorLength> aosoa( "my_aosoa",
                                                               num_tuple );
 
+    // Create a mirror view of the aosoa on the host for accessing it legally
```

Comment:
```suggestion
    // Create a mirror of the AoSoA on the host (this will not make a new copy if the memory is already allocated on the host, but will if it is in device memory) in order to print out the data.
```